scroll_menu.py

```diff
@@ -56,62 +103,13 @@ def main() -> None:
     )
 
     layout = Layout(container=root_container)
-
-    # Key bindings.
-    kb = KeyBindings()
-
-    @kb.add("c-c")
-    def exit_(event: str) -> None:
-        get_app().exit()
-
-    kb.add("tab")(focus_next)
-    kb.add("s-tab")(focus_previous)
-
+    kb = setup_key_bindings()
     # Create and run application.
     application = Application(
         layout=layout, key_bindings=kb, full_screen=True, mouse_support=True
     )
     application.run()
```

Comment:
Since we're not going to be running `scroll_menu.py` locally to invoke the application, this should be hooked up to the text_editor code.